isis/src/base/apps/spkwriter/tsts/apollo15_mixed/Makefile

```diff
@@ -12,8 +12,8 @@ commands: $(BASES)
 #  Only use 12 digits in the output of tabledump due to precision issues
 $(BASES): $(OUTPUT)/apollo15_mixed.bsp
 	$(CP) $(INPUT)/$@.cub $(OUTPUT)
-	spiceinit from=$(OUTPUT)/$@.cub ckrecon=true cksmithed=true spk=$(OUTPUT)/apollo15_mixed.bsp > /dev/null;
-	tabledump from=$(OUTPUT)/$@.cub to=$(OUTPUT)/$@.txt \
+	spiceinit $(ISISROOT)/TestPreferences from=$(OUTPUT)/$@.cub ckrecon=true cksmithed=true spk=$(OUTPUT)/apollo15_mixed.bsp > /dev/null;
+	tabledump $(ISISROOT)/TestPreferences from=$(OUTPUT)/$@.cub to=$(OUTPUT)/$@.txt \
```

Comment:
You should use `$(TSTARGS)`, that is what we use in the category tests for this.

Reply:
Thanks, forgot about that. Also changed the other two tests in the same way